Sources/Unbox.swift

```diff
@@ -407,6 +407,13 @@ public class Unboxer {
         return try Unboxer(dictionary: dictionary, context: context).performCustomUnboxingWithClosure(closure)
     }
     
+    /// Perform custom unboxing on an array of dictionaries, executing a closure with a new Unboxer for each one, discarding elements producing errors
+    public static func performFlattenedCustomUnboxingWithArray<T>(array: [UnboxableDictionary], context: Any? = nil, closure: Unboxer throws -> T?) -> [T] {
```

Comment:
Instead of introducing a new method for this, is it possible to use the same `allowInvalidElements:` parameter convention like was introduced for other arrays here: https://github.com/JohnSundell/Unbox/pull/50?